warehouse/query-core/src/test/java/datawave/query/jexl/visitors/LiteralNodeSubsetVisitorTest.java

```diff
@@ -0,0 +1,122 @@
+package datawave.query.jexl.visitors;
+
+import com.google.common.collect.Multimap;
+import datawave.query.jexl.JexlASTHelper;
+import org.apache.commons.jexl2.parser.ASTJexlScript;
+import org.apache.commons.jexl2.parser.ParseException;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+public class LiteralNodeSubsetVisitorTest {
+    
+    private final Set<String> expectedLiterals = new HashSet<>(Arrays.asList("FOO", "BAR", "BAZ", "ABCD", "1234", "1b3d"));
+    
+    @Test
+    public void testSingleLiteralWithMatch() throws ParseException {
+        String queryString = "FOO == '1234'";
+        ASTJexlScript script = JexlASTHelper.parseJexlQuery(queryString);
+        Multimap<String,String> literals = LiteralNodeSubsetVisitor.getLiterals(expectedLiterals, script);
+        
+        Assert.assertEquals(1L, literals.size());
```

Comment:
Nit: size() is returning an integer I think so this is probably auto-converting the size() to a long unnecessarily. We could just do `assertEquals(1, literals.size())`

Reply:
Changed as suggested.